person.rb

```diff
@@ -32,6 +32,10 @@ def add_rental(date, book)
     Rental.new(date, self, book)
```

Comment:
- [ ] It would be better if we are able to run the app following the instructions in the README file, however, I'm unable to run the app successfully. Kindly add `ruby main` to successfully run the app

Reply:
@addod19 Okay. I will do that.

Reply:
@addod19 I read your review about not seeing the ID. The I'd of the user that you created the rental for was used . Kindly use that and you shall have the rentals.